core/execution/reference_price_moves.go

```diff
@@ -72,7 +75,15 @@ func (m *Market) checkForReferenceMoves(
 	}
 
 	// now we can start all special order repricing...
-	orderUpdates = m.repriceAllSpecialOrders(ctx, changes, orderUpdates)
+	if err == nil {
+		orderUpdates = m.repriceAllSpecialOrders(ctx, changes, orderUpdates)
+	} else {
+		cancels := m.liquidity.UndeployLPs(ctx, orderUpdates)
+		orderUpdates, err = m.updateAndCreateLPOrders(ctx, nil, cancels, nil)
+		if err != nil {
+			panic(err) // this should not be possible
```

Comment:
Can we use m.log.Panic("error message", logging.Error(err))
```suggestion
			m.log.Panic("could not get best static prices and unable to cancel lp orders", logging.Error(err), logging.MarketID(m.mkt.GetID())) // this should not be possible
```

Reply:
will update